AY2021S2-CS2103-T14-1/tp
# Add list alias command

## What this does
This PR
1. adds list alias command

Closes #120 

## How to test
1. Launch SunRez.
2. Delete all previous aliases (either delete the pref file or unalias them).
3. Execute the `aliases` command. You should see a no alias created yet message. 
4. Create some aliases.
5. Execute the `aliases` command. You should all your aliases listed.
